### DISCUSSION TITLE
Where are specs for v9 components?

### DISCUSSION THREAD

**Discussion by sjwilczynski** [microsoft/fluentui#29143] [Q&A] [answered]
Hey folks, I was looking for a spec for Button v9 component and when I checked README there is supposedly link to one: 
![image](https://github.com/microsoft/fluentui/assets/19153565/415b65bd-4ffe-49e3-ab5d-a28aafaa9b16)
However, such file doesn't exist. For Table component for example, spec file is in root of react-table package. Should other components have specs added to the repo as well? Or are they somewhere else?

**Comment by ValentinaKozlova** [comment, marked as answer]
Hello @sjwilczynski, for V9 components you might want to use a documentation.
Here's for Button:
https://react.fluentui.dev/?path=/docs/components-button-button--docs
